lib/nahmii-provider.js

```diff
@@ -41,16 +43,21 @@ class NahmiiProvider extends ethers.providers.JsonRpcProvider {
      * @param {string} apiAppSecret - nahmii API app-secret
      * @param {string} nodeUrl - url to an ethereum node to connect to
      * @param {string|number} network - a known ethereum network name or ID
+     * @param {EthereumAddress} [operator] - The address of the nahmii operator
      */
-    constructor(nahmiiDomain, apiAppId, apiAppSecret, nodeUrl, network) {
+    constructor(nahmiiDomain, apiAppId, apiAppSecret, nodeUrl, network, operator = null) {
         super(nodeUrl, network);
 
+        if (operator && !(operator instanceof EthereumAddress))
+            throw new TypeError('Invalid argument - operator not an EthereumAddress');
+
         _nahmiiDomain.set(this, nahmiiDomain);
         _appId.set(this, apiAppId);
         _appSecret.set(this, apiAppSecret);
         _nahmii.set(this, new NahmiiRequest(nahmiiDomain, () => {
             return this.getApiAccessToken();
         }));
+        _operator.set(this, (operator || null));
     }
```

Comment:
Redundant default value for operator?

Reply:
No, its forcing for instance "undefined" to become "null"